<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SalesPlotDialog</class>
 <widget class="QDialog" name="SalesPlotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>График продаж</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblInfo">
     <property name="text">
      <string>Укажите промежуток времени по которому будет отрисовываться график продаж.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ltDates">
     <item>
      <widget class="QLabel" name="lblFrom">
       <property name="text">
        <string>с </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="deFrom">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblTo">
       <property name="text">
        <string> по </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="deTo">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnPlot">
       <property name="text">
        <string>Построить график</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGraphicsView" name="gvPlot">
     <property name="renderHints">
      <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="chbDeliveries">
       <property name="text">
        <string>Поставки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chbSales">
       <property name="text">
        <string>Продажи</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnShowSums">
       <property name="text">
        <string>Отображать суммы</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
